ChangeLog.md

```diff
@@ -4,9 +4,12 @@
 
  - Disable sessions since we don't use them.
  - Add query parameter `includeRawRejectReason` to the `accTransactions` query.
- - health query checks connections to database and GRPC, and that the last final
+ - Health query checks connections to database and GRPC, and that the last final
    block is less than `health-tolerance` seconds old. `health-tolerance` is 300
    seconds unless an alternative value is chosen at startup.
+ - Add filtering options `blockTimeFrom`, `blockTimeTo`, `blockRewards`, 
+  `finalizationRewards`, `bakingRewards`, and  `onlyEncrypted` to the 
+  `accTransactions` query. Removed the `includeRewards` option.
```

Comment:
Unfortunately we cannot remove the existing option. We need to maintain backwards compatibility.

We should instead add a new endpoint `v1/accTransactions` that supports the new options, or support all the options in the existing endpoint.